securesystemslib/metadata.py

```diff
@@ -0,0 +1,82 @@
+"""Dead Simple Signing Envelope
+"""
+
+from typing import Any, List
+
+from securesystemslib import exceptions
+from securesystemslib import formats
+from securesystemslib.signer import Signature
+from securesystemslib.util import b64dec, b64enc
+
+
+class Envelope:
+    """DSSE Envelope.
+
+    Attributes:
+        payload: Arbitrary byte sequence of Serialized Body
+        payloadType: string that identifies how to interpret payload
+        signatures: List of Signature and GPG Signature
+
+    """
+
+    payload: bytes
+    payloadType: str
```

Comment:
We generally follow [PEP 8 and thus use snake case for variable names](https://peps.python.org/pep-0008/#function-and-variable-names). Can you please change this here and elsewhere in this PR, even though the spec uses camelCase for this field?

```suggestion
    payload_type: str
```